pallets/delegation/src/tests.rs

```diff
@@ -442,17 +445,26 @@ fn check_add_and_revoke_delegations() {
 		assert_noop!(
 			Delegation::revoke_root(
 				Origin::signed(account_hash_bob.clone()),
-				H256::from_low_u64_be(999)
+				H256::from_low_u64_be(999),
+				1
 			),
 			Error::<Test>::RootNotFound
 		);
 		assert_noop!(
-			Delegation::revoke_root(Origin::signed(account_hash_bob), id_level_0),
-			Error::<Test>::UnauthorizedRevocation
+			Delegation::revoke_root(Origin::signed(account_hash_bob), id_level_0, 1),
+			Error::<Test>::UnauthorizedRevocation,
+		);
+		let res =
```

Comment:
Why not the usual check?

```suggestion
		assert_noop!(Delegation::revoke_root(Origin::signed(account_hash_alice.clone()), id_level_0, 0), Error::<Test>::ExceededRevocationBounds);
```

Reply:
forgot that this exists. f8f7e8c